packages/angular/package.json

```diff
@@ -39,6 +39,7 @@
     "@nrwl/jest": "*",
     "@nrwl/linter": "*",
     "@schematics/angular": "~12.0.0",
-    "jasmine-marbles": "~0.6.0"
+    "jasmine-marbles": "~0.6.0",
+    "webpack-merge": "4.2.1"
```

Comment:
The version here should be `5.7.3`, with the Angular upgrade we now use Webpack 5.

Reply:
Good spot